diff -Naur Redis-Client-0.012/MANIFEST Redis-Client-0.012.expireat/MANIFEST
--- Redis-Client-0.012/MANIFEST	2012-01-02 23:14:57.000000000 +0100
+++ Redis-Client-0.012.expireat/MANIFEST	2012-03-06 02:19:03.000000000 +0100
@@ -37,7 +37,7 @@
 t/cmd_eval.t
 t/cmd_exists.t
 t/cmd_expire.t
-t/cmd_expiteat.t
+t/cmd_expireat.t
 t/cmd_flushall.t
 t/cmd_flushdb.t
 t/cmd_get.t
diff -Naur Redis-Client-0.012/lib/Redis/Client.pm Redis-Client-0.012.expireat/lib/Redis/Client.pm
--- Redis-Client-0.012/lib/Redis/Client.pm	2012-01-02 23:14:57.000000000 +0100
+++ Redis-Client-0.012.expireat/lib/Redis/Client.pm	2012-03-06 02:19:03.000000000 +0100
@@ -24,7 +24,7 @@
         DEL         => [ 1, undef ],
         EXISTS      => 1,
         EXPIRE      => 2,
-        EXPITEAT    => 2,
+        EXPIREAT    => 2,
         KEYS        => 1,
         MOVE        => 2,
         OBJECT      => [ 1, undef ],
diff -Naur Redis-Client-0.012/t/cmd_expireat.t Redis-Client-0.012.expireat/t/cmd_expireat.t
--- Redis-Client-0.012/t/cmd_expireat.t	1970-01-01 01:00:00.000000000 +0100
+++ Redis-Client-0.012.expireat/t/cmd_expireat.t	2012-03-06 02:19:03.000000000 +0100
@@ -0,0 +1,22 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use utf8;
+use lib 't';
+
+use Test::More;
+
+# ABSTRACT: Tests for the Redis EXPIREAT command.
+
+use_ok 'RedisClientTest';
+
+my $redis = RedisClientTest->server;
+done_testing && exit unless $redis;
+
+isa_ok $redis, 'Redis::Client';
+
+# TODO: write tests!
+
+done_testing;
+
diff -Naur Redis-Client-0.012/t/cmd_expiteat.t Redis-Client-0.012.expireat/t/cmd_expiteat.t
--- Redis-Client-0.012/t/cmd_expiteat.t	2012-01-02 23:14:57.000000000 +0100
+++ Redis-Client-0.012.expireat/t/cmd_expiteat.t	1970-01-01 01:00:00.000000000 +0100
@@ -1,22 +0,0 @@
-#!/usr/bin/env perl
-
-use strict;
-use warnings;
-use utf8;
-use lib 't';
-
-use Test::More;
-
-# ABSTRACT: Tests for the Redis EXPITEAT command.
-
-use_ok 'RedisClientTest';
-
-my $redis = RedisClientTest->server;
-done_testing && exit unless $redis;
-
-isa_ok $redis, 'Redis::Client';
-
-# TODO: write tests!
-
-done_testing;
-
